doc/api/util.md

```diff
@@ -330,6 +330,24 @@ changes:
 The `util.inspect()` method returns a string representation of `object` that is
 primarily useful for debugging. Additional `options` may be passed that alter
 certain aspects of the formatted string.
+`util.inspect()` will use the constructor's name and/or `@@toStringTag` to make an
```

Comment:
Empty line before this sentence to make this a new paragraph.

Reply:
it's not meant to be a new paragraph, newline is just for readability while editing